README.md

```diff
@@ -56,7 +56,7 @@ let crunker = new Crunker();
 crunker
   .fetchAudio('/voice.mp3', '/background.mp3')
   .then((buffers) => crunker.mergeAudio(buffers))
-  .then((merged) => crunker.export(merged, 'audio/mp3'))
+  .then((merged) => crunker.export(merged, 'audio/mp3', true))
```

Comment:
Perhaps we should use object destructing with defaults for opts.

```ts
crunker.export(merged, { format: 'audio/mp3', stereo: true} )
```

Reply:
yes, but that would be a **breaking change** and all working code with `export(audioBuffer, fileType)` will stop working

Reply:
Unless we build in backwards compatibility for it, and use a function overload for typings and mark the original behaviour as deprecated?

Reply:
I managed to get rid of this option so we don't change the API anymore!